docs/reference/continuous-export.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Continuous export
+parent: Reference
+nav_order: 10
+has_children: false
+---
+# Continuous Export
+
+You can configure a lakeFS repository to store the latest of a branch on an external
+object store using its native paths and access methods.  This allows clients and tools
+read-only access to repository objects without any lakeFS-specific configuration.
+Initially, only AWS S3 is supported.
+
+For instance, the contents `lakefs://example@master` might be stored on
+`s3://company-bucket/example/latest`.  Clients entirely unaware of lakeFS could use that
+base URL to access latest files on `master`.  Clients aware of lakeFS can continue to use
+the lakeFS S3 endpoint to access repository files on `s3://example/master`, as well as
+other versions and uncommitted versions.
+
+## Configuration
+
+To configure continuous export for branch `master`, run
+
+```shell
+lakefs branch export --branch master --start --export-path s3://company-bucket/path/to/export/ [--export-status-path s3://company-bucket/path/to/status]
+```
+
+To stop continuous export, run
+
+```shell
+lakefs branch export --branch master --stop
+```
+
+This works any number of your branches, not only `master`.
+
+If you omit the `--start` flag, the branch tip will be exported just once, at that time.
+
+```shell
+lakefs branch export --branch experiment1 --export-path s3://company-bucket/path/to/experiment/ [--export-status-path s3://company-bucket/path/to/experiment_status]
+```
+
+## Operation
+
+When continuous export is enabled for a branch its current state is exported to S3.  From
+then on, every commit or merge operation to that branch will be exported.  The commit or
+merge operation on the branch returns as soon as export starts.  Multiple commits to the
+same exported branch may cause some otherwise-overlapping exports to be skipped.
+Eventually the last export will appear.  This is in line with "eventual consistency" of
+S3.
+
+The export process runs for a while.  When it completes successfully it writes an empty
```

Comment:
I think maybe we should start with something simpler.

I suggest using Delta's convention of a hidden folder (maybe `_lakefs_status/`?) underwhich we place a single file per exported commit once that export finished successfully in its entirety.

this allows for a simple way for consumers to poll for completion - for example, a simple Airflow sensor that polls for the file's existence before running a job. we can use the actual commit ids as file names and get an actual ordered log of exported commits by listing this prefix (i.e a `_lakefs_status/master@50.json` file naming convention.

inside each file, place the list of exported paths that finished successfully (with etag, size, etc).